packages/ui/src/components/collectibles/native/collectible-text.native.tsx

```diff
@@ -1,14 +1,14 @@
 import { useEffect, useState } from 'react';
 
-// import DomPurify from 'dompurify';
-import { Text } from '../../../../native';
+import { Text } from '../../text/text.native';
 import { CollectibleCardLayout } from './collectible-card-layout.native';
 
 interface CollectibleTextProps {
   src: string;
+  size?: number;
 }
 
-export function CollectibleText({ src }: CollectibleTextProps) {
+export function CollectibleText({ src, size = 200 }: CollectibleTextProps) {
   const [content, setContent] = useState<string | null>(null);
 
   useEffect(() => {
```

Comment:
Probably worth turning this into a query, instead of `effect` + `state` + `throw`. `fetch` doesn't throw on HTTP errors, but regardless, we wouldn't want to crash the page/app for this.

Reply:
Thinking about this now and that maybe this should be done in the `collectibles` service. What do you think @alexp3y ?

Reply:
I'm going to leave this as is for now as I think we should fetch inscription text in the services.